lib/pip.js

```diff
@@ -275,12 +275,16 @@ async function installRequirements(targetFolder, pluginInstance) {
 
       dockerCmd.push('docker', 'run', '--rm', '-v', `${bindPath}:/var/task:z`);
       if (options.dockerSsh) {
+        const homePath = require('os').homedir();
+        const sshKeyPath =
+          options.dockerPrivateKey || `${homePath}/.ssh/id_rsa`;
+
         // Mount necessary ssh files to work with private repos
         dockerCmd.push(
           '-v',
-          `${process.env.HOME}/.ssh/id_rsa:/root/.ssh/id_rsa:z`,
```

Comment:
Could you check if that change to `os.homedir()` could be causing these issues with permissions on runner? AFAIK it still takes the result from `process.env.HOME` but nothing else in this PR looks like the cause of this problem and all other PRs are passing without problems 😕